.docker/autoline/_docker-compose.dev.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.8"
+
+# Using `node_modules` mounting pattern described here:
+# https://burnedikt.com/dockerized-node-development-and-mounting-node-volumes/#prevent-hosts-node_modules-from-being-mounted
+
+services:
+  autoline_mariadb:
+    container_name: autoline_mariadb
+    build:
+      context: ../../
+      dockerfile: database/Dockerfile
+    restart: on-failure
+    ports:
+      - "3307:3306"
+    volumes:
+      - autoline_mariadb_data:/var/lib/mysql
+    env_file:
+      - ../../.env/backend.env
+    networks:
+      - autoline_backend_network
+
+  autoline_backend:
+    container_name: autoline_backend
+    depends_on:
+      - autoline_mariadb
+    build:
+      context: ../../
+      dockerfile: backend/Dockerfile.dev
+    restart: unless-stopped
+    ports:
+      - "3001:3001"
+    volumes:
+      - ../../shared:/shared/
+      - exclude_shared:/shared/node_modules/
+      - ../../backend:/backend/
+      - exclude_backend:/backend/node_modules/
+    env_file:
+      - ../../.env/backend.env
+    environment:
+      - FRONTEND_URL=http://autoline_frontend:3000
+    networks:
+      - autoline_backend_network
+      - autoline_frontend_network
+
+  autoline_frontend:
+    container_name: autoline_frontend
+    depends_on:
+      - autoline_backend
+    build:
+      context: ../../
+      dockerfile: frontend/Dockerfile.dev
+    restart: unless-stopped
+    ports:
+      - "3000:3000"
+    volumes:
+      - ../../shared:/shared/
+      - exclude_shared:/shared/node_modules/
+      - ../../frontend:/frontend/
+      - exclude_frontend:/frontend/node_modules/
+    env_file:
+      - ../../.env/frontend.env
+    environment:
+      - VITE_API_ORIGIN_URL=http://autoline_backend:3001/api/v1
+    networks:
+      - autoline_backend_network
+      - autoline_frontend_network
+
+volumes:
+  autoline_mariadb_data:
+    driver: local
+
+networks:
+  autoline_backend_network:
+    driver: bridge
+  autoline_frontend_network:
+    driver: bridge
```

Comment:
Do we need this file? Does it work on Windows?

Reply:
No. This is old version of the file. I did not mention that and committed it, sorry.